kavm/src/kavm/pyk_utils.py

```diff
@@ -36,12 +36,30 @@ def tvalue(value: Union[str, int, bytes]) -> KInner:
     else:
         raise TypeError()
 
+def tvalue_elem(value: Union[str, int, bytes]) -> KInner:
+    if type(value) is int:
+        return intToken(value)
+    elif type(value) is OnComplete:
+        return intToken(int(value))
+    elif type(value) is str:
+        return stringToken(value)
+    elif type(value) is bytes:
+        return stringToken(value.decode())
+    else:
+        raise TypeError()
```

Comment:
This should not have to be a separate function from `tvalue()`. The issue is that when I add a bytes value to a TValueList, the base64 decoding done by `tvalue()` puts an encoded string into, e.g., the application args cell, where it will not be in the correct form for the semantics. However, this decoding is used elsewhere, so I can't just remove it.